pkg/ignore/oktetoignore.go

```diff
@@ -15,32 +15,64 @@ package ignore
 
 import (
 	"bufio"
-	"errors"
 	"fmt"
 	"io"
 	"os"
 	"regexp"
 	"strings"
 	"sync"
 
+	"github.com/moby/patternmatcher"
 	"github.com/moby/patternmatcher/ignorefile"
 )
 
 const RootSection = "__DEFAULT___"
 
-type Ignore struct {
+type OktetoIgnorer struct {
 	sections map[string]string
 	mu       sync.RWMutex
 }
 
-func (i *Ignore) Get(section string) (data string) {
+// BuildOnly returns a new OktetoIgnorer that will only consider the root
```

Comment:
Didn't we say that the okteto ignore shouldn't have a build section. I think it's best to remove it so it doesn't cause confusion